src/BlazorWasmSdk/Tasks/ComputeBlazorBuildAssets.cs

```diff
@@ -200,6 +200,10 @@ private static void ApplyUniqueMetadataProperties(ITaskItem candidate)
                         candidate.SetMetadata("AssetTraitName", "BlazorWebAssemblyResource");
                         candidate.SetMetadata("AssetTraitValue", "runtime");
                     }
+                    if (string.Equals(candidate.GetMetadata("ResolvedFrom"), "{HintPathFromItem}", StringComparison.Ordinal))
+                    {
+                        candidate.SetMetadata("OriginalItemSpec", candidate.ItemSpec);
+                    }
```

Comment:
We need this to account for the fact that the assembly was resolved using HintPath, otherwise static web assets thinks the `.csproj` is the original representation of the item and will try to copy that to the output folder.

Will be adding some more unit tests in this area